### PR TITLE
Add mute rules

### LINKED ISSUE #216: Custom word blacklist/filter for question submissions
one of the things i liked about curiouscat that i've not been able to find on any other similar site, is the ability to have a custom word blacklist/filter that if included in the question, would prevent the question being sent or send the question into the void never ot be seen again instead of into someone's inbox. At most, sites have had a checkbox that would filter questions based on their undisclosed filter system, often based around profanity. It'd be nice to be able to filter out/prevent questions being submitted that mention certain words determined by the user.

### PR DESCRIPTION
**Pull request by raccube**
Closes #216

### CODE REVIEW

**Review by pixeldesu — APPROVED** [review]
LGTM :+1:

File: app/views/user/edit_mute.haml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+= render 'settings/muted', qr_svg: @qr_svg, recovery_code_count: @recovery_code_count
```
Comment (nilsding) [inline app/views/user/edit_mute.haml]: qr_svg?

**Review by nilsding — CHANGES_REQUESTED** [review]
small stuff, also maybe it makes sense to not use an incremental (by 1) id for these?

File: app/views/user/edit_mute.haml (hunk 1 above)
Comment (raccube) [reply app/views/user/edit_mute.haml]: Wow, I blindly the template from `edit_security`.

**Review by nilsding — APPROVED** [review]

**Pull request merged**